### PR TITLE
Add Limelight to Hardware overview and Vision Processing Section

### PR DESCRIPTION
**Pull request by bhjelstrom**
Trying to be informative - let me know what I can change to make this work. Thanks!

### CODE REVIEW

File: source/docs/software/vision-processing/limelight/what-is-limelight.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+        .. code-block:: python
+
+            from networktables import NetworkTables
+
+            table = NetworkTables.getTable("limelight")
+            tx = table.getNumber('tx',None)
+            ty = table.getNumber('ty',None)
+            ta = table.getNumber('ta',None)
+            ts = table.getNumber('ts',None)
```
Comment (katzuv) [inline source/docs/software/vision-processing/limelight/what-is-limelight.rst]: Add a space after the `,` inside the function call.

File: source/StartAutobuilding.bat [hunk 2]
```diff
@@ -0,0 +1 @@
+sphinx-autobuild . _build/html
```
Comment (Daltz333) [inline source/StartAutobuilding.bat]: What is the point of this file?

**Comment by Daltz333** [comment]
Any update, or is this stale?

**Comment by sciencewhiz** [comment]
The control system hardware page is for items that are (or have been) in the KOP or are required to be approved (like the speed controllers). Adding the limelight there isn't appropriate. A link on the third party library page would be more appropriate. The getting started stuff should be on the limelight website.

**Comment by Daltz333** [comment]
Closing as stale. Feel free to reopen if relevant.

**Pull request closed without merging**